Configuration/commands.json

```diff
@@ -245,6 +245,16 @@
             },
             "category": "Fun 🎪"
         },
+        "dogfact": {
+            "usage": "",
```

Comment:
The command appears to have usage other than blank. Change this to `[<fact-count>]` or similar.